Examples/UIExplorer/UIExplorerApp.android.js

```diff
@@ -38,6 +39,8 @@ const UIExplorerNavigationReducer = require('./UIExplorerNavigationReducer');
 const UIExplorerStateTitleMap = require('./UIExplorerStateTitleMap');
 const URIActionMap = require('./URIActionMap');
 
+UIManager.setLayoutAnimationEnabledExperimental(true);
```

Comment:
I don't think so, I added a LayoutAnimation example in UIExplorer directly and I think the animation example app is still part of UIExplorer anyway.